docs/openDAQ-streaming.md

```diff
@@ -392,15 +392,20 @@ There are some example of signal descriptions in a separate document.
       <contains at least one signal member description>
     },
     "tableId": <string>,
+    ["valueIndex": <uint64>],
     ["relatedSignals": [
         { "type" : <relation type>, "signalId" : <id of the related signal> } 
       ]
-    ]
+    ],
     ["interpretation": {}]
   }
 }
 ~~~~
 
+- `tableId`: Id of the table the signal belongs to
+- `valueIndex`: Optional, and relevant only if domain signal of the table follows an implicit linear rule.

```

Comment:
The same would be the case for any implicit signal, not only implicit linear, although the only real case I can think of is linear time. Just to make this a bit more general, I would suggest removing "linear" from the sentence above.
You could add an example "such as a implicit linear time signal".